SatHunter/SatInfoView.swift

```diff
@@ -50,18 +67,14 @@ struct SatInfoView: View {
               "Max el:"
             )
             Spacer()
-            Text("\(String(format: "%.0f", maxEl!)) deg")
+            Text("\(String(format: "%.0f", maxEl!)) °")
           }
         }
         HStack {
-          Text("Your grid:")
+          Text("My Grid:")
           Spacer()
           Text(userGrid)
         }
-        HStack {
```

Comment:
is there a better way to indicate this? The decision of presenting local time is only for my convenience and I'm not sure if it's a surprise to other users.